addons/editor/functions/fnc_bi_curatorWaypointPlaced.sqf

```diff
@@ -18,11 +18,23 @@
  */
 #include "script_component.hpp"
 
-if (GVAR(unitRadioMessages) == 2) exitWith {};
-
 params ["", "_group", "_waypointID"];
 
-[
-    leader _group,
-    ["CuratorWaypointPlaced", "CuratorWaypointPlacedAttack"] select (waypointType [_group, _waypointID] == "DESTROY")
-] call BIS_fnc_curatorSayMessage;
+private _leader = leader _group;
+private _waypoint = [_group, _waypointID];
+private _waypointType = waypointType _waypoint;
+
+if (GVAR(unitRadioMessages) != 2) then {
+    [
+        _leader,
+        ["CuratorWaypointPlaced", "CuratorWaypointPlacedAttack"] select (_waypointType == "DESTROY")
+    ] call BIS_fnc_curatorSayMessage;
+};
+
+if (_waypointID == 1) then {
+    [_group, 0] setWaypointPosition [getPos _leader, 0];
```

Comment:
`getPos` returns AGLS, `setWaypointPosition` expects AGL or Object. Can pass the leader directly here.

Do we want to force exact position here? If so, then this should use ASL with a -1 radius.

Reply:
Yes, let's go for the -1 option.